velox/functions/prestosql/aggregates/tests/ApproxMostFrequentTest.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/file/FileSystems.h"
+#include "velox/functions/prestosql/aggregates/tests/AggregationTestBase.h"
+
+namespace facebook::velox::aggregate::test {
+namespace {
+
+template <typename T>
+struct ApproxMostFrequentTest : AggregationTestBase {
+  ApproxMostFrequentTest() {
+    noSpill_ = true;
+  }
+
+  std::shared_ptr<FlatVector<int>> makeGroupKeys() {
+    return makeFlatVector<int>(3, [](auto row) { return row; });
```

Comment:
nit: maybe use a constant for the number of groups.

Reply:
Let's keep it simple in tests :D